plugin.video.easyengineering/default.py

```diff
@@ -0,0 +1,23 @@
+#   Copyright (C) 2018 Lunatixz
+#
+#
+# This file is part of Easy Engineering TV.
+#
+# Easy Engineering TV is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Easy Engineering TV is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Easy Engineering TV.  If not, see <http://www.gnu.org/licenses/>.
+
+# -*- coding: utf-8 -*-
+
+#entrypoint
+if __name__ == '__main__':
+    from resources.lib import easytv
```

Comment:
make sure you test this in leia/current nightly

Reply:
I'm running last week's nightly, anything in particular change?

Reply:
Yeap merged on tuesday: https://github.com/xbmc/xbmc/pull/13814
To understand what changed you can have a read here: https://github.com/xbmc/generator-kodi-addon/pull/38